core/src/net/sf/openrocket/rocketcomponent/Parachute.java

```diff
@@ -9,24 +9,31 @@
 
 public class Parachute extends RecoveryDevice {
 	private static final Translator trans = Application.getTranslator();
-	
-	public static double DEFAULT_CD = 0.8;
-	
+	private final double defaultDiameter = 0.3;
```

Comment:
I would change this to DEFAULT_DIAMETER, since it's a constant, not a real variable

Reply:
Changed.